src/website/layouts/partials/header.pug

```diff
@@ -43,8 +43,17 @@ header.chi-header.-inverse.docs-header.-z--20
         .chi-btn__content
           i.chi-icon.-sm--2.icon-menu
       a.chi-brand.docs-header__logo(href=`${rootPath}`, aria-label='CenturyLink')
-        img(src='https://assets.ctl.io/images/centurylink-logo-white.svg', alt='CenturyLink')
-        span.-sr--only CenturyLink
+        svg(viewBox='0 0 204 41', xmlns='http://www.w3.org/2000/svg', aria-labelledby='CenturyLink', role='img')
```

Comment:
`aria-labeledby` is not used correctly. It's value should be [one or more element IDs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute). From mozilla developer docs: "This idea is similar to how the "for" attribute is used to link a LABEL to an INPUT."